CanDIG/mohccn-synthetic-data
# Add DONOR_ALL, DONOR_NULL, use import instead of subprocess

* Changed to using clinical_etl import in csv_to_ingest.py
* split code into multiple files for more modularity
* Changed DONOR_000 to DONOR_NULL, made all 'NULL' objects have NULL in id where possible
* Changed DONOR_011 to DONOR_ALL_01 - has valid values for every possible field, but also adheres to conditional logic, e.g. the donor is not deceased so there is no date of death
* Added DONOR_ALL_02 - a 'complete' donor that is deceased.
* Regenerated csvs

To test:
* run src/csv_to_ingest.py with a dataset of choice
* Ensure no validation errors are thrown (there will be a lot of warnings)
* try to ingest the resulting json `raw_data_map.json` into a running instance of the stack
